src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionEditForm.tag

```diff
@@ -0,0 +1,107 @@
+<%@ tag description="instructorFeedbacks - form for editing feedback question" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ tag import="teammates.common.util.FieldValidator" %>
+<%@ tag import="teammates.logic.core.Emails.EmailType" %>
+<%@ tag import="teammates.common.datatransfer.FeedbackParticipantType" %>
+
+<%@ taglib tagdir="/WEB-INF/tags/instructor/feedbackEdit" prefix="feedbackEdit" %>
+
+<%@ attribute name="fqForm" type="teammates.ui.template.FeedbackQuestionEditForm" required="true"%>
+
+<form class="form-horizontal form_question" role="form" method="post"
+    action="${fqForm.action}"
+    id="form_editquestion-${fqForm.question.questionNumber}" name="form_editquestions"
+    onsubmit="tallyCheckboxes(${fqForm.question.questionNumber})"
+    ${ fqForm.questionHasResponses ? "editStatus=\"hasResponses\"" : "" }>
```

Comment:
Use `'editStatus="hasResponses"' : ''` instead.
The standard for strings in ternary operator in EL seems to be single quotes.